src/elasticodbc/dlg_specific.h

```diff
@@ -52,6 +52,7 @@ extern "C" {
 #endif /* UNICODE_SUPPORT */
 
 #define INI_SERVER "host"
+#define INI_SERVER_ALT "server"
```

Comment:
In PR #7 I renamed this so that it's INI_HOST "host" and INI_SERVER "server"

Reply:
Merged in new changes